### PR TITLE
Sdmext 591 display attachments specific to a repository

### PR DESCRIPTION
**Pull request by rashmiangadi11**
Attachments specific to repository should be displayed based on repositoryId in query

### CODE REVIEW

**Review by yashmeet29 — APPROVED** [review]
LGTM

File: sdm/src/main/java/com/sap/cds/sdm/constants/SDMConstants.java [hunk 1]
```diff
@@ -1,6 +1,10 @@
 package com.sap.cds.sdm.constants;
 
 public class SDMConstants {
+  private SDMConstants() {
```
Comment (maheshsooryambylu) [inline sdm/src/main/java/com/sap/cds/sdm/constants/SDMConstants.java:4]: Is this constructor required ?

**Review by maheshsooryambylu — COMMENTED** [review]
Looks good

File: sdm/src/main/java/com/sap/cds/sdm/constants/SDMConstants.java (hunk 1 above)
Comment (rashmiangadi11) [reply sdm/src/main/java/com/sap/cds/sdm/constants/SDMConstants.java:4]: yes its a sonar issue fix

**Pull request merged**